modules/bootkube/resources/bootkube.sh

```diff
@@ -11,20 +11,32 @@ echo "Rendering Kubernetes core manifests..."
     --config=/assets/kco-config.yaml \
     --output=/assets
 
-echo "Rendering TNC manifests..."
+echo "Rendering MCO manifests..."
 
 # shellcheck disable=SC2154
 /usr/bin/docker run \
     --user 0 \
     --volume "$PWD:/assets:z" \
-    "${tnc_operator_image}" \
-    --config=/assets/tnco-config.yaml \
-    --render-bootstrap=true \
-    --render-output=/assets/tnc-bootstrap
+    "${machine_config_operator_image}" \
+    bootstrap \
+    --etcd-ca=/assets/tls/etcd-client-ca.crt \
+    --root-ca=/assets/tls/root-ca.crt \
+    --config-file=/assets/manifests/cluster-config.yaml \
+    --dest-dir=/assets/mco-bootstrap \
+    --images-json-configmap=/assets/manifests/machine-config-operator-01-images-configmap.yaml
 
 mkdir -p /etc/kubernetes/manifests/
-cp "$PWD/tnc-bootstrap/tectonic-node-controller-pod.yaml" /etc/kubernetes/manifests/
-cp "$PWD/tnc-bootstrap/tectonic-node-controller-config.yaml" /etc/kubernetes/tnc-config
+mkdir -p /etc/mcc/bootstrap/
+mkdir -p /etc/ssl/mcs/
+# Bootstrap MachineConfigController uses /etc/mcc/bootstrap/manifests/ dir to
+# 1. read the controller config rendered by MachineConfigOperator
+# 2. read the default MachineConfigPools rendered by MachineConfigOperator
+# 3. read any additional MachineConfigs that are needed for the default MachineConfigPools.
+cp -r "$PWD/mco-bootstrap/manifests" /etc/mcc/bootstrap/manifests
+# /etc/ssl/mcs/tls.{crt, key} are locations for MachineConfigServer's tls assets.
+cp "$PWD/tls/machine-config-server.crt" /etc/ssl/mcs/tls.crt
+cp "$PWD/tls/machine-config-server.key" /etc/ssl/mcs/tls.key
+cp "$PWD/mco-bootstrap/machineconfigoperator-bootstrap-pod.yaml" /etc/kubernetes/manifests/
```

Comment:
This doesn't appear to be a local file.  Is it created by the `bootstrap` rendering call above?  I think this would be less brittle if we were able to volume-mount a handful of target directories, and then the installer could put its generated assets in the right places on its own.  But if this is the only such file, than handling it explicitly here is probably an acceptable short-term solution.

Reply:
We need special logic for some operators that we need for bootstrapping the cluster:

https://github.com/openshift/installer/blob/77cc46f9a5e279303e0908f1b25707e5fcd555a1/modules/bootkube/resources/bootkube.sh#L16-L27

https://github.com/openshift/installer/blob/77cc46f9a5e279303e0908f1b25707e5fcd555a1/modules/bootkube/resources/bootkube.sh#L92

Everything else is already present in the correct directory `<something>/manifests`

Reply:
> Is it created by the bootstrap rendering call above? 

yes